jayacados/ReactionTime
# Feature: Add Beaker page component

# Feature: Beaker Page Component

## Description:
- Add createRoot, packageTest, fallbackTest, suspenseTest cards
- Style card components
- Add MaterialUI as a dependency

## Changes I Made:
- Commented out .git/hooks to remove git commit obstacle

## How to Test:
- Run sandbox of code on codesandbox.io to verify code in working condition
- Test component once react routing has been established by other developers
